src/lib/project-saver-hoc.jsx

```diff
@@ -78,11 +80,11 @@ const ProjectSaverHOC = function (WrappedComponent) {
             }
             return this.storeProject(this.props.reduxProjectId)
                 .then(() => {
+                    // there is nothing we expect to find in response that we need to check here
```

Comment:
What does this comment mean? It kind of reads like "These are not the droids you're looking for".

Reply:
Clarified